tests/orm/utils/test_node.py

```diff
@@ -23,3 +23,48 @@ def test_load_node_class_fallback():
     with pytest.warns(UserWarning):
         loaded_class = load_node_class('__main__.SubData.')
     assert loaded_class == Data
+
+
+# Test for Remove_prefix
+
+
+def test_load_node_class_with_node_prefix():
+    """Test the behavior of load_node_class with node prefix."""
+    # Test node prefix removal
+    with pytest.warns(UserWarning):
```

Comment:
Can you please match the specific warning, the same way you do in `test_load_node_class_with_data_prefix`? Same on line 39 below